vms/platformvm/vm_test.go

```diff
@@ -119,6 +122,15 @@ var (
 	errMissing = errors.New("missing")
 )
 
+func init() {
+	for _, key := range keys {
+		nodeBytes := key.PublicKey().Address()
```

Comment:
Because of the `TestGetStake` test in `vms/platformvm/service_test.go` which does some checks against genesis content. I think we should eventually clean it up, just chose not to do it in this PR

Reply:
Could we put a TODO?

Reply:
Done